wiki/markdown/query.md

```diff
@@ -192,3 +192,22 @@ Examples of this endpoint using curl
 
     curl -X GET -H "Content-type: application/json" localhost:39835/queryMissedNotifications?SubscriptionName=Subscription_1
```

Comment:
"queryMissedNotifications",, shouldn't all occurence of the entrypoint be renamed with the new entrypoint name?

Reply:
Not yet. It is still queryMissedNotifications in master.